spot-client/src/index.js

```diff
@@ -143,6 +144,17 @@ history.listen(location => {
     store.dispatch(routeChanged(location));
 });
 
+/**
+ * Persisting query params.
+ */
+const rawParamsObject = {};
+
+for (const [ key, value ] of queryParams.entries()) {
+    rawParamsObject[key] = value;
+}
+
+store.dispatch(setStartParams(rawParamsObject));
```

Comment:
You could also do this in the `createStore` call so you don't need the action, unless it will happen soon where the start params could be changed after bootstrap.

Reply:
Ok, I do a follow up commit